<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClickForm</class>
 <widget class="QWidget" name="ClickForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>361</width>
     <height>540</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="9" column="0">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>放弃</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>H</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="winHLE"/>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <widget class="QCheckBox" name="apCheck">
       <property name="text">
        <string>使用ap药</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QPushButton" name="setEnd">
       <property name="text">
        <string>设定完毕</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>测试物品</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>W</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="winWLE"/>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>放在窗口上2秒来获取窗口句柄</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>鼠标点击后等待时间再做下步动作</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="mouseDely">
         <property name="text">
          <string>200</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="winYLE"/>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="winXLE"/>
       </item>
      </layout>
     </item>
     <item row="3" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>检测失败时等待时间后再次检测</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="checkDely">
         <property name="text">
          <string>1800</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="winHWID">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>检测成功后等待时间后将鼠标移过去点击</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="waitDely">
         <property name="text">
          <string>2200</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="8" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>248</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>等待AP不足界面出现</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="apbuzuLe">
         <property name="text">
          <string>7000</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="page_2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>从选关卡开始</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>从自动再战开始</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>从检测游戏物品开始</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QTextEdit" name="textEdit"/>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="stopBT">
       <property name="text">
        <string>停止</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
